azure-pipelines.vrt-pr.yml

```diff
@@ -1,7 +1,7 @@
 pr:
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28432&commitId=e124715778abd4883829c01ba2429fbbd38d4057&env=prod&clientType=FLUENTUI)


  <details><summary><b>Field</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.RadioGroup.default.chromium.png|55125| Changed|
|Field.Select.default.chromium.png|42295| Changed|

</details>


  <details><summary><b>Radio Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.group-vertical - RTL.default.chromium.png|20410| Changed|
|Radio Converged.group-disabled.default.chromium.png|20399| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|20400| Changed|
|Radio Converged.group-vertical.default.chromium.png|20409| Changed|

</details>


  <details><summary><b>Select Converged</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Select Converged.Appearance- outline (default).hover.chromium.png|6568| Changed|
|Select Converged.Appearance- underline.focused.chromium.png|6541| Changed|
|Select Converged.Custom Icon.default.chromium.png|6617| Changed|
|Select Converged.Disabled.default.chromium.png|6431| Changed|
|Select Converged.Invalid- filled-darker.default.chromium.png|6760| Changed|
|Select Converged.Invalid- filled-darker.hover.chromium.png|6760| Changed|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|6888| Changed|
|Select Converged.Appearance- filled-lighter.default.chromium.png|6561| Changed|
|Select Converged.Invalid- outline.default.chromium.png|6573| Changed|
|Select Converged.Appearance- outline (default).default.chromium.png|6568| Changed|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|7241| Changed|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|6561| Changed|
|Select Converged.Invalid- outline.hover.chromium.png|6573| Changed|
|Select Converged.Invalid- underline.focused.chromium.png|6541| Changed|
|Select Converged.Size- large.default.chromium.png|9848| Changed|
|Select Converged.Invalid- outline.focused.chromium.png|6542| Changed|
|Select Converged.With appearance override.default.chromium.png|13779| Changed|
|Select Converged.Size- small.default.chromium.png|6496| Changed|
|Select Converged.Invalid- underline.default.chromium.png|6567| Changed|
|Select Converged.Invalid- filled-lighter.default.chromium.png|6888| Changed|
|Select Converged.Invalid- underline.hover.chromium.png|6567| Changed|
|Select Converged.With value.default.chromium.png|6643| Changed|
|Select Converged.Appearance- filled-darker.default.chromium.png|6443| Changed|
|Select Converged.Appearance- underline.default.chromium.png|6566| Changed|
|Select Converged.Appearance- filled-darker.focused.chromium.png|7103| Changed|
|Select Converged.Appearance- underline.hover.chromium.png|6566| Changed|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|7241| Changed|
|Select Converged.Appearance- filled-darker.hover.chromium.png|6443| Changed|
|Select Converged.Appearance- outline (default).focused.chromium.png|6542| Changed|
|Select Converged.Invalid- filled-darker.focused.chromium.png|7103| Changed|

</details>




<div id="vrtCommentfluent-ui"/>